.travis.yml

```diff
@@ -10,4 +10,11 @@ compiler:
 
 script:
   - sed -i 's|TOPCARATDIR=|TOPDIR=/home/travis/build/lbfm-rwth/carat|g' Makefile
+  - sed -i 's|CFLAGS = -g -Wall -DDIAG1|CFLAGS = -coverage -g -Wall -DDIAG1|g' Makefile
   - make
+  - cd bin; ln -s `ls -d */` carat
+  - cd ../tst; ./run_all.sh
+
+after_script:
+  - gcov functions/*/*.c*
```

Comment:
Why the `.c*` instead of `.c`? Are there C++ files in there?

Reply:
nope:) ^C ^V was it.